src/server/components/dashboard/controllers.ts

```diff
@@ -153,12 +155,34 @@ export async function listsController(req: Request, res: Response, next: NextFun
       isNewUser,
       lists: listsWithFormattedDates(lists as ListsForDashboard[]),
       csrfToken: getCSRFToken(req),
+      dashboardBoxes: calculateDashboardBoxes(lists as ListsForDashboard[]),
     });
   } catch (error) {
     next(error);
   }
 }
 
+function calculateDashboardBoxes(lists: ListsForDashboard[]) {
+  return [calculateAdminDashboardBox(lists)];
+}
+
+function calculateAdminDashboardBox(lists: ListsForDashboard[]) {
+  const adminBox = {
+    name: "administrators",
+    text: "All lists have administrators",
+    cssClass: "green",
+  };
+
+  const listsWithNoAdmins = lists.filter((list) => list.admins === 0);
+
+  if (listsWithNoAdmins.length > 0) {
+    adminBox.text = `${listsWithNoAdmins.length} lists have no administrators`;
+    adminBox.cssClass = "red";
```

Comment:
ideally we would use something like "warning" or "danger" and "success", so the server is colour blind, and it's up to the view/css to figure our how to render a "warning". 

although I'm pretty sure we've shortcutted in a few places and use colours, so I'll let you decide what you use. but for best practice it falls under separation of concerns. and partly under usability. "red" should not be the only thing that indicates that there is a problem.